src/auth/impersonated.ts

```diff
@@ -157,11 +158,8 @@ export class Impersonated extends OAuth2Client implements IdTokenProvider {
 
   /**
    * Refreshes the access token.
-   * @param refreshToken Unused parameter
    */
-  protected async refreshToken(
-    refreshToken?: string | null
```

Comment:
Is this a breaking change?

Reply:
Technically, yes, however since its introduction a few years ago I think this would cause more confusion for customers rather than cleaning it up:
- https://github.com/googleapis/google-auth-library-nodejs/pull/1207

I included this change in the current PR as it was causing a lint issue.